File: Examples/Application_With_Pods_File/TestMerchAppExample_Obj-C/TestMerchAppExample_Obj-C/ViewController.m
//
//  ViewController.m
//  TestMerchAppExample_Obj-C
//
//  Created by PLMAC-A1278-C1MLJUH1DTY3 on 4/23/15.
//  Copyright (c) 2015 opus. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

@synthesize homePayBtn, phonePayBtn;

- (void)viewDidLoad {
    [super viewDidLoad];
    
    // Do any additional setup after loading the view, typically from a nib.
}

/* ------------------- UI placement ----------------------------- */

-(void)placeUIData{
    
}

/* ------------------- Main screen data ------------------------- */

-(IBAction)homeBtnSelected:(id)sender{
   
  /*  _homeController = [[HomeViewController alloc] initWithNibName:nil bundle:nil];
    UIStoryboard *storyboard = self.storyboard;
    _homeController = [storyboard instantiateViewControllerWithIdentifier:@"HomeviewController"];
    [self presentViewController:_homeController animated:YES completion:nil];
    */
    
    _menuScreen =[[MenuScreen alloc] initWithNibName:nil bundle:nil];
    UIStoryboard *storyboard = self.storyboard;
    _menuScreen = [storyboard instantiateViewControllerWithIdentifier:@"MenuScreen"];
    [self presentViewController:_menuScreen animated:YES completion:nil];
}


/* ------------------- Back pressed ----------------------------- */

-(IBAction)backPressed:(UIStoryboardSegue *)seque{
    [self.navigationController popViewControllerAnimated:YES];
}

/* ------------------- Home purchase work ---------------------- */






/* ------------------- Phone purchase work --------------------- */



#pragma mark Real SDK calling


#pragma mark End

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Examples/TestMerchAppExample_Swift/TestMerchAppExample_Swift/ViewController.swift
//
//  ViewController.swift
//  TestMerchAppExample_Swift
//
//  Created by PLMAC-A1278-C1MLJUH1DTY3 on 4/23/15.
//  Copyright (c) 2015 opus. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        //        var dataDictonary: [String: String] = ["applicationPrimaryAccountNumber": "4111111111111111", "applicationExpirationDate":"181231" ,  "transactionAmount":"1499", "cardholderName":"Bill Gates"]
        //
        //
        //        var optTokenObj:OPTestFakeAppleToken = OPTestFakeAppleToken()
        //        optTokenObj.prepareRequestFakeApplePayToken(dataDictonary)
        
        
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func backPressed(_ x:UIStoryboardSegue) {
        self.navigationController?.popViewController(animated: true)
    }
}
